.github/workflows/pr.yml

```
@@ -26,3 +26,31 @@ jobs:
 
       - name: Run spotless check
         run: mvn -B spotless:check
+
+  frontend:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)